sections/main-article.liquid

```diff
@@ -52,17 +52,20 @@
         {%- when 'share' -%}
           <div class="article-template__social-sharing page-width page-width--narrow" {{ block.shopify_attributes }}>
             <share-button class="share-button" {{ block.shopify_attributes }}>
-              <button class="share-button__button button button--tertiary hidden">
+              <button class="share-button__button hidden">
                 {% render 'icon-share' %}
                 {{ 'general.share.share' | t }}
               </button>
               <details>
-                <summary class="share-button__button button button--tertiary">
+                <summary class="share-button__button">
```

Comment:
Back when I first implemented this component, Scott did an accessibility review that included some suggestions to use properties like `role="button"`, `aria-expanded` and `aria-controls` for when JS is enabled.

I didn't have the time yet to work on this issue and it may also affect other components in the theme using the `DetailsDisclosure` component, but just letting you know in case you wanna address it here.

You can find it under PR `#420` in our previous dawn fork. I'll send you a link.